CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
         - **.PATCH**: Pull Request Title (PR Author) [PR Number](Link to PR)
 -->
 #### 6.10
+- **.17**: Add 'b' binding to to jump to bookmark table. (ds2606) [#1305](https://github.com/preservim/nerdtree/pull/1305)
```

Comment:
We are not doing a new version for each PR anymore, just remove this change log. For now, I'll use the PR titles to write multi-bulletin change logs.